src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_params.py

```diff
@@ -3,10 +3,26 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 #---------------------------------------------------------------------------------------------
 
+import platform
+
 from azure.cli.core.commands import register_cli_argument
 from azure.cli.core.commands.parameters import (
     name_type,
     resource_group_name_type)
+from azure.cli.core._util import CLIError
+
+def _get_default_install_location(exeName):
+    system = platform.system()
+    if system == 'Windows':
+        install_location = 'C:\\Program Files\\{}.exe'.format(exeName)
```

Comment:
Also, `os.environ.get('ProgramFiles')` instead as then you don't get a KeyError from the dict and can check for None on the result and not set install location if you get none back from the call to os.environ.